specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/storage.json

```diff
@@ -5530,6 +5540,9 @@
           }
         }
       },
+      "required": [
```

Comment:
If we add "targetRegion" support in the future, will "targetSkuName" still be required? (Since with before parameter description, seems the 2 parameters can't be used together.)

Please note it will be breaking change to change a required parameter to optional, and will need Azure’s Breaking Change Review Board review. See details in last page of [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)